src/PanZoom.tsx

```diff
@@ -79,4 +83,32 @@ const PanZoomWithContext = (props: PanZoomProps, apiRef: PanZoomProps['apiRef'])
   </PanZoomProvider>
 );
 
-export default forwardRef(PanZoomWithContext);
+const PanZoomWithRef = forwardRef(PanZoomWithContext);
+
+const PanZoomWithCover: React.FC<PanZoomProps> = ({
+  cover,
+  zoomMax,
+  width,
+  height,
+  ...props
+}) => {
+  const { ref, size, zoom } = useCover(cover);
+
+  return (
+    <div style={{ width, height }}>
+      <PanZoomWithRef
+        {...props}
+        {...size}
+        ref={ref}
+        boundary={{}}
```

Comment:
could be just boolean `boundary`

Reply:
probably, but in that case you should probably change the TS type